### PR TITLE
Issue 4

### PR DESCRIPTION
**Pull request by bdeanindy**
Code reviewer checklist to resolve Issue #4 

- [ ] Remove duplicate and out-of-date LICENSE.txt file
- [ ] Removed full-text of MaxMinds License from within Webhooks docs, and added reference inline (PRIMARY ISSUE RESOLUTION)
- [ ] Per-Legal: Created new [3rd Party Licenses](/docs/3RD_PARTY_LICENSES.md)
- [ ] Per-Legal: Added MaxMinds license to [3rd Party Licenses](/docs/3RD_PARTY_LICENSES.md)
- [ ] Per-Legal: Include appropriate reference to 3rd Party License in main license file
- [ ] (SCOPE CREEP...but needed done) Removed the deployment directions in [README](README.md)

### CODE REVIEW

**Comment by bdeanindy** [comment]
@CHromek could you please review this for me so I may merge it?

File: services/webhooks_api.md [hunk 1]
```diff
@@ -17,22 +17,8 @@ The following are key operational details:
 | auth_token | string | Authentication token to present in the X-MessageSystems-Webhook-Token header of POST requests to target | no | Use this token in your target application to confirm that data is coming from the Webhooks API. <br />example: `5ebe2294ecd0e0f08eab7690d2a6ee69`|
 | events     | array  | Array of event types this webhook will receive | yes | Use the Webhooks Events endpoint to list the available event types.<br />example: `["delivery", "injection", "open", "click"]`|
 
-## Copyrights
+__**Sparkpost Webhooks use MaxMinds software [MaxMinds License](/docs/3RD_PARTY_LICENSES.md)**__
```
Comment (CHromek) [inline services/webhooks_api.md]: By including the file here, do you need a github account to be able to see this 3RD_PARTY_LICENSES.md file?

Comment (thuxtable1) [inline services/webhooks_api.md]: Ben - this comment looks good to me- but just to follow-up on Chere's question - is a github account required?

**Comment by bdeanindy** [comment]
@thuxtable1 No, you can view the changed files associated with this commit here:

https://github.com/SparkPost/sparkpost-api-documentation/pull/7/files

This PR is out of date and I'll have to pull/resolve the conflicts.

@richleland Do you want me to also include the link for the other reference of MaxMinds as I've done with this sir?

File: services/webhooks_api.md (hunk 1 above)
Comment (richleland) [inline services/webhooks_api.md]: This repo is public so anyone will be able to view the file without having a github account.

File: docs/3RD_PARTY_LICENSES.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# 3rd Party Licenses
+
+## MaxMinds
```
Comment (richleland) [inline docs/3RD_PARTY_LICENSES.md]: MaxMind - singular

File: services/webhooks_api.md (hunk 1 above)
Comment (richleland) [inline services/webhooks_api.md]: * Need capital P - SparkPost
* Webhooks doesn't need to be capitalized
* MaxMind - singular

File: LICENSE [hunk 3]
```diff
@@ -200,3 +200,5 @@ Apache License
    See the License for the specific language governing permissions and
    limitations under the License.
 
+   3rd Party Licenses:
+   SparkPost Webhooks utilize MaxMinds software, view the [MaxMinds License](/docs/3RD_PARTY_LICENSES.md)
```
Comment (richleland) [inline LICENSE]: * Webhooks doesn't need to be capitalized
* MaxMind - singular

**Comment by richleland** [comment]
@bdeanindy yeah would be great to update across the board. I've added some minor copy edits in comments, other than that LGTM once you pull and resolve conflicts.

File: LICENSE (hunk 3 above)
Comment (thuxtable1) [inline LICENSE]: I've reviewed them and they LGTM... Theo

**Comment by colestrode** [comment]
I'm going to resolve and merge this. I'm in the Message Events docs, so I'll remove the copyright info there and replace with what @bdeanindy is adding for webhooks.

**Comment by colestrode** [comment]
This was merged into the DEV docs and will be released publicly soon

**Pull request closed without merging**